prow/manifests/overlays/metal3/config.yaml

```diff
@@ -138,11 +138,7 @@ branch-protection:
                   [
                     "test-centos-e2e-integration-main",
                     "test-ubuntu-integration-main",
-                    "metal3-bmo-e2e-test",
                   ]
-            release-0.5:
-              required_status_checks:
-                contexts: ["metal3-bmo-e2e-test"]
```

Comment:
Not sure if we should really remove these. I marked the job as not optional below but I'm not sure if that means it will show a status context. If it doesn't show the context we should add these back.